sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -378,6 +379,23 @@ def run_Flatten(self, transform_node):
           PropertyNames.ENCODING: step.encoding,
           PropertyNames.OUTPUT_NAME: PropertyNames.OUT}])
 
+  def apply_WriteToBigQuery(self, transform, pcoll):
```

Comment:
this is only overriding the batch mode for streaming we can keep using the streaming api so the ptransfrom should be okay.